base_force_record_noupdate/models/res_config_settings.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2024 Camptocamp SA
+# @author Italo LOPES <italo.lopes@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+
+    _inherit = "res.config.settings"
+
+    set_default_models = fields.Boolean(
+        string="Set default models",
+        help="Set the default models that should have noupdate=True.",
+    )
+    models_force_noupdate = fields.Many2many(
+        "ir.model",
+        string="Models to force noupdate=True",
```

Comment:
a many to many that becomes a param? Why?
This should probably be a m2m on the company?
All this complexity can be trashed. What is the pro of keeping it?